indico/modules/events/client/js/reviewing/components/TimelineItem.jsx

```diff
@@ -67,6 +68,10 @@ export default function TimelineItem({revision, isLastRevision, state}) {
                     <div dangerouslySetInnerHTML={{__html: revision.commentHtml}} />
                   </>
                 )}
+                {/* TODO: Check whether the current user is submitter */}
+                {isLastRevision && state.name === 'needs_submitter_confirmation' && (
```

Comment:
should use the enum (InitialRevisionState or FinalRevisionState) depending on which state we are using here